plugins/modules/inventory_workflow_manager.py

```diff
@@ -2282,7 +2281,7 @@ def get_interface_from_ip(self, device_ip):
 
                 if response:
                     interface_id = response[0]["id"]
-                    self.log("Fetch Interface Id for device '{0}' successfully !!".format(device_ip))
+                    self.log("Fetch Interface Id for device '{0}' successfully !!".format(device_ip), "DEBUG")
```

Comment:
self.log("Successfully retrieved Interface Id '{0}' for device '{1}'.".format(interface_id, device_ip), "DEBUG")

Reply:
updated.